ui/roll20_character_sheet_0.2.html

```diff
@@ -1409,22 +1405,57 @@
   }
   on('clicked:undotoproll', undoTopRoll);
 
-  function epRoll() {
-    setAttrs({
-      pendingchat: "!EPRoll",
-    });
-  }
-  on('clicked:epRoll', epRoll);
-
-  function epRollWithModifier() {
-    getAttrs(["roll_modifier"], (values) => {
+  const EP_ROLL_OPTIONS = [{
```

Comment:
I consolidated the various epRoll functions into one array of options.
This worked well.